client/build_code/webpack_common.js

```diff
@@ -58,9 +58,14 @@ const get_rules = ({ target_ie11, language, is_prod_build }) => {
       use: js_module_loader_rules,
       sideEffects: true,
     },
+    {
+      //NIVO_TODO: Temporary workaround for bug from Nivo. Will need to delete this as it gets fixed in newer versions
+      test: /react-spring/,
+      sideEffects: true,
+    },
     {
       // node modules that specifically require transpilation...
-      include: /node_modules\/(graphiql|graphql-language-service-.*|codemirror-graphql|codemirror)/,
+      include: /node_modules\/(graphiql|graphql-language-service-.*|codemirror-graphql|codemirror|d3-scale|@nivo\/bar|@nivo\/circle-packing|@nivo\/core|@nivo\/line|@nivo\/pie|)/,
```

Comment:
Could be `@nivo\/*`? OR are there some nivo packages that don't need transpilation?

Reply:
I should actually check these individual packages